packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -12,47 +12,59 @@ const chatMessageStyles: ComponentSlotStylesInput<
   ChatMessageProps & ChatMessageState,
   ChatMessageVariables
 > = {
-  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    display: 'inline-block',
-    paddingLeft: v.padding,
-    paddingRight: v.padding,
-    paddingTop: pxToRem(8),
-    paddingBottom: pxToRem(10),
-    borderRadius: v.borderRadius,
-    border: v.border,
-    color: v.color,
-    backgroundColor: p.mine ? v.backgroundColorMine : v.backgroundColor,
-    maxWidth: v.width,
-    position: 'relative',
-    wordBreak: 'break-word',
-    wordWrap: 'break-word',
-    outline: 0,
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const marginOffset = pxToRem(100)
 
-    ...((v.hasMention || v.isImportant) && {
-      '::before': {
-        content: '""',
-        backgroundColor: v.hasMention ? v.hasMentionColor : v.isImportantColor,
-        height: '100%',
-        left: '0',
-        position: 'absolute',
-        top: '0',
-        width: pxToRem(3),
-        borderBottomLeftRadius: 'inherit',
-        borderTopLeftRadius: 'inherit',
-      },
-    }),
+    return {
+      display: 'inline-block',
+      position: 'relative',
+
+      marginLeft: p.mine ? marginOffset : 0,
+      marginRight: !p.mine ? marginOffset : 0,
+      maxWidth: `calc(100% - ${marginOffset})`,
+      minWidth: marginOffset,
```

Comment:
Would introduce the marginOffset as a variable, but it is used as minWidth as well, which will mean maybe we need variables... Just thinking out loud :) Maybe it's better if it stays as it is..